### DISCUSSION TITLE
Blank screens on Android

### DISCUSSION THREAD

**Discussion by williambuchanan2** [dotnet/maui#15325] [General]
Is anyone else experiencing blank screens on Android.

As I navigate between pages, often all I get is a blank white page. Sometimes the content shows if I pull my finger down the screen but most of the time I have to hit the back button then try again. I would say around 50% of the time I get this.

I have now tried it on 2 different devices, one of which is a new device.

It doesn't happen on the emulator.

**Comment by Xyncgas** [comment]
*Stuffs I am complaining is more about api pattern, although imo sometimes they cause problem like this due to people internally in microsoft can trip over these anti-patterns

me too, on Desktop before app launch, sometimes blank screen exist a while sometimes shorter, it can be due to something locking or lagging or UI update is somehow forgotten by the framework as [er usual microsoft quality

I do not understand exactly why, I am hoping it's nothing to do with me using threads, the stuffs on dotnet with configure await and marshaling and resume on different thread and synchronization context etc to me is a big gotcha that makes things not work by default if you do something that's not their prefer ways of doing - - often I just wish things work because 99% I am writing codes that shouldn't be dealing with these things (Even today I gotta choose ComponentBase.InvokeAsync (StateHasChanged) in blazor from another thread instead of just calling that component's StateHasChanged method
